Missions/Missions/Games Tab/BalanceGameViewController.swift

```diff
@@ -224,13 +247,22 @@ class BalanceGameViewController: UIViewController {
         self.scoreView.addSubview(scoreLabel)
         
         // game view
+        self.gameView.presentScene(gameScene)
         self.gameView.addSubview(self.horizontalCrossHair)
         self.gameView.addSubview(self.verticalCrossHair)
         
         self.gameView.addSubview(self.leftBall)
         self.gameView.addSubview(self.rightBall)
         self.gameView.addSubview(self.singleBall)
         
+//        if let view = self.gameView as SKView? {
```

Comment:
If this code block isn't going to be used, be sure to remove it. Otherwise, label it with a `TODO:` in a line above the code block